src/index.ts

```diff
@@ -12,7 +12,8 @@ const equivalents = [
   'no-unused-expressions',
   'no-useless-constructor',
   'quotes',
-  'space-before-function-paren'
+  'space-before-function-paren',
+  'semi'
```

Comment:
```suggestion
  'semi',
  'space-before-function-paren'
```

Just trying to keep this sorted.